dags/analyzer_helper/discord/discord_transform_raw_members.py

```diff
@@ -24,20 +23,22 @@ def transform_member(self, member: dict) -> dict:
         """
         Transform a single member's data to the guildmember structure.
         """
-        guild_member = {
-            "id": int(member.get("discordId")),
-            "is_bot": member.get("isBot", False),
-            "left_at": member.get("deletedAt"),
-            "joined_at": member.get("joinedAt"),
-            "options": {
-                # "username": member.get("username"),
-                # "avatar": member.get("avatar"),
-                # "roles": member.get("roles"),
-                # "discriminator": member.get("discriminator"),
-                # "permissions": member.get("permissions"),
-                # "global_name": member.get("globalName"),
-                # "nickname": member.get("nickname"),
+        guild_member = (
```

Comment:
Are you passing just one member to this transform method or a list of raw members? 

The parameter in the method header (`raw_members`) does have a type of list, so the `get` method would not work and you need to do a for loop for the given list of raw members (if you're passing a list of raw members).